File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.java
class Solution {
    private int[] cnt = new int[26];
    public String solution(String s) {
        StringBuilder ret = new StringBuilder();
        for(char c : s.toCharArray()) cnt[c - 'a']++;
        for(int i = 0; i < cnt.length; i++){
            if(cnt[i] == 1) ret.append((char)(i + 'a'));
        }
        return ret.toString();
    }
}

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
from collections import Counter
def solution(s):    
    cnt = [0] * 26
    for c in s:
        cnt[ord(c)-ord('a')] += 1
    return ''.join(chr(i +ord('a')) for i in range(len(cnt)) if cnt[i] == 1)
        